### ISSUE TITLE
Modifying Links 

### ISSUE THREAD

**Issue by zackify** [sstur/react-rte#47]
Is there a way that I can add `target="blank"` to all <a /> tags before calling `toString('html')`? It would be cool if there was an easy way to modify tags being outputted.

**Comment by sstur** [comment]
I agree there should be an optional second parameter to this method, an object `options` that should expose features like this. Actually the toString directly calls [draft-js-export-html](https://github.com/sstur/draft-js-export-html) so could you open an issue there and as soon as the feature gets in there, the fix for this here will be trivial.

**Comment by sstur** [comment]
Note: There is already an issue over at https://github.com/sstur/draft-js-export-html/issues/10 in which we talked about "a feature that allows you to pass in a set of attributes you want to support"

**Comment by skolmer** [comment]
I have created a PR that allows you to pass the `options` argument to `toString` #111
